src/platform/backends/shared/base_snapshot.h

```diff
@@ -84,6 +87,12 @@ inline std::string multipass::BaseSnapshot::get_comment() const
     return comment;
 }
 
+inline std::string multipass::BaseSnapshot::get_parent_name() const
+{
+    const std::shared_lock lock{mutex};
+    return parent ? parent->get_name() : "--";
```

Comment:
Simpler in what sense? 

I initially had an empty string, I am logging the parent in `VM::take_snapshot` and I wanted to use the same form as specified for the CLI. But I suppose we can duplicate the string and still leave it to the CLI to decide how to present it.

Reply:
OK yeah, you're right that this is a presentation thing and should be up to the client. Do you mind if I update it in the upcoming PR I am working on though? Just to avoid a few rebases...